package.json

```diff
@@ -41,7 +42,6 @@
     "perf:debug": "cross-env PERF=true gulp perf:debug --debug",
     "postinstall": "node ./scripts/postinstall.js",
     "preinstall": "node ./scripts/use-yarn-please.js",
-    "prelint": "yarn satisfied && yarn syncpack:list",
```

Comment:
this was very generic, what `prelint` means and in what context is not self explanatory